test/unit/src/common/containers/t_with-repository.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import withRepository from '../../../../../src/common/containers/with-repository';
+
+const middlewares = [ thunk ];
+const mockStore = configureMockStore(middlewares);
+
+// Dummy test component to wrap
+const Dummy = () => <span>Dummy</span>;
+
+import { SET_GITHUB_REPOSITORY } from '../../../../../src/common/actions/create-snap';
+
+describe('WithRepository', () => {
+  const PARAMS = {
+    owner: 'anowner',
+    name: 'aname'
+  };
+  const REPO = {
+    fullName: 'anowner/aname'
+  };
+
+  let store;
+
+  let wrapper;  // WithRepository rendered wrapper
+  let instance; // WithRepository wrapper instance
+  let wrapped;  // wrapped Dummy component
+
+  const setupDummy = (repository = REPO, params = PARAMS) => {
```

Comment:
Can this helper function be reworked to return `wrapper` instead of writing it to a global? It would make it possible to move the function to the bottom of the file, making the body of the unit test easier to read.